### DISCUSSION TITLE
Please output segmentation mask from Ops.Libs.Mediapipe.MpPoseTracking

### DISCUSSION THREAD

**Discussion by tgd-git** [cables-gl/cables_docs#318] [Feature Requests] [answered]
The op has config pins for segmentation mask but no output of the mask. would be super nice to have.

![image](https://user-images.githubusercontent.com/2562637/185181046-4e6322e2-ade5-4134-bf33-cc9697301d57.png)

https://google.github.io/mediapipe/solutions/pose.html#segmentation_mask

**Comment by steam0r** [comment, marked as answer]
this is now available on dev.cables.gl. thanks for supporting us in the development of this feature!

**Reply by marioepsley** [reply]
any demo, how do you access the mask?
